articles/step-by-step-on-how-to-user-nodejs-repl-tutorial/index.md

```diff
@@ -0,0 +1,203 @@
+### Introduction
+Node.js `Read -Evaluate-Print-Loop` is an easy-to-use command-line tool for processing Node.js expressions.
+This shell captures the user's JavaScript code inputs, interprets and evaluates the result of this code, displays the result to the screen, and repeats the process till the user quits the shell.  
+
+It's therefore, important to note that this tool does not require file creation to write codes. It comes with ready to use Node.js development environment.
+
+### Prerequisites
+This tutorial requires:
+1. Node.js Installed in your development local machine.
+2. Basic knowledge of JavaScript.
+
+### Getting started with REPL
+Enter the following in the terminal:
```

Comment:
Before getting started with the tutorial, state the objectives of what is going to be done. This helps readers understand what they're going to learn.